ports/gsasl/portfile.cmake

```
@@ -0,0 +1,30 @@
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://ftp.gnu.org/gnu/gsasl/gsasl-${VERSION}.tar.gz"
```

Maybe it would be better to use tagged archives from https://gitlab.com/gsasl/gsasl instead?

---

You can change it in another PR.

---

I think an HTTPS download is fine. It's `vcpkg_from_git` that creates pain for caching systems.

---

But I would strongly suggest to add a mirror. I did this for similar ports.

```suggestion
    URLS "https://ftp.gnu.org/gnu/gsasl/gsasl-${VERSION}.tar.gz"
         "https://www.mirrorservice.org/sites/ftp.gnu.org/gnu/gsasl/gsasl-${VERSION}.tar.gz"
```

---

This is exactly the reason I suggested to use GitLab: I strongly believe its uptime and response time are better than the ftp.
@talregev, I just want to help. No need to be aggressive again.